doc/ref_model/chapters/appendix-a.md

```diff
@@ -70,6 +71,22 @@ Without arguing for or against SR-IOV, CNTT provides the following anecdotes whi
 End of policies.
 
 <a name="A.6"></a>
+## A.6 VNF Design and Implementation Requirements
```

Comment:
```suggestion
## A.6 VNF Design and Implementation Guidelines
```

Reply:
@kedmison These are Requirements. I think we should include the ONAP "[VNF or PNF Requirements Documentation](https://onap.readthedocs.io/en/latest/submodules/vnfrqts/requirements.git/docs/index.html)"

Reply:
I believe the CNTT should focus on standardizing the VNF<->NFVI interface.  I believe it's OK to make observations about VNF design that best takes advantage of this VNF-NFVI interface, but requirements about the process for building the VNF, or the contents of it, are probably best left to other forum like ONAP, etc.  

The other reason is that if it is made mandatory here how is it going to be testable?  Is the RC going to mandate these requirements, and then be required to execute tests for them? If not, then the requirement is not really a 'requirement' and may serve to weaken the RM document instead.